EstateAdministrationUI/Areas/Estate/Controllers/OperatorController.cs

```diff
@@ -0,0 +1,148 @@
+﻿namespace EstateAdministrationUI.Areas.Estate.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Security.Claims;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using BusinessLogic.Models;
+    using Common;
+    using Factories;
+    using Microsoft.AspNetCore.Authentication;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Models;
+    using Newtonsoft.Json;
+    using Services;
+    using Shared.Logger;
+    
+    [ExcludeFromCodeCoverage]
+    [Authorize]
+    [Area("Estate")]
+    public class OperatorController : Controller
+    {
+        #region Fields
+
+        /// <summary>
+        /// The API client
+        /// </summary>
+        private readonly IApiClient ApiClient;
+
+        /// <summary>
+        /// The view model factory
+        /// </summary>
+        private readonly IViewModelFactory ViewModelFactory;
+
+        #endregion
+
+        #region Constructors
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HomeController" /> class.
+        /// </summary>
+        /// <param name="apiClient">The API client.</param>
+        /// <param name="viewModelFactory">The view model factory.</param>
+        public OperatorController(IApiClient apiClient,
+                                  IViewModelFactory viewModelFactory)
+        {
+            this.ApiClient = apiClient;
+            this.ViewModelFactory = viewModelFactory;
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Creates the merchant.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        [HttpGet]
+        public async Task<IActionResult> CreateOperator(CancellationToken cancellationToken)
+        {
+            CreateOperatorViewModel viewModel = new CreateOperatorViewModel();
+
+            return this.View("CreateOperator", viewModel);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateOperator(CreateOperatorViewModel viewModel,
+                                                        CancellationToken cancellationToken)
+        {
+            // Validate the model
+            if (this.ValidateModel(viewModel))
+            {
+                String accessToken = await this.HttpContext.GetTokenAsync("access_token");
+
+                CreateOperatorModel createOperatorModel = this.ViewModelFactory.ConvertFrom(viewModel);
+
+                // All good with model, call the client to create the operator
+                CreateOperatorResponseModel createOperatorResponse =
```

Comment:
Codacy found an issue: [Remove the unused local variable 'createOperatorResponse'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6070534)